src/sentry/options/defaults.py

```diff
@@ -1305,24 +1305,29 @@
 register("span_descs.bump-lifetime-sample-rate", default=0.25, flags=FLAG_AUTOMATOR_MODIFIABLE)
 # Decides whether artifact bundles asynchronous renewal is enabled.
 register("sourcemaps.artifact-bundles.enable-renewal", default=0.0, flags=FLAG_AUTOMATOR_MODIFIABLE)
-# Enables reporting status of queues for backpressure management.
-register(
-    "backpressure.monitor_queues.enable_status", default=False, flags=FLAG_AUTOMATOR_MODIFIABLE
-)
-# Enables checking queue health in consumers for backpressure management.
-register("backpressure.monitor_queues.enable_check", default=False, flags=FLAG_AUTOMATOR_MODIFIABLE)
+
+# === Backpressure related runtime options ===
+
+# Enables monitoring of services for backpressure management.
+register("backpressure.monitoring_enabled", default=False, flags=FLAG_AUTOMATOR_MODIFIABLE)
```

Comment:
_nit_: No strong opinion, just looking at other examples `backpressure.monitoring.enabled` could be a neatly structured alternative.